.idea/copyright/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```

Comment:
@whatzerrname we have to resolve conflicts on .gitignore above maybe to remove this file